### DISCUSSION TITLE
vscode: SQL auto complete/suggest

### DISCUSSION THREAD

**Discussion by code1704** [microsoft/vscode-discussions#838] [Extension Development QnA] [answered]
Is this any way for sql auto complete in VSCode? vscode built in support or vscode extension?

e.g. 

```
SELECT use[suggest r_id, ..] FROM users
```

**Comment by gjsjohnmurray** [comment, marked as answer]
I help maintain [SQLTools](https://marketplace.visualstudio.com/items?itemName=mtxr.sqltools) which might help you.
